polling_stations/settings/base.py

```diff
@@ -312,3 +312,9 @@
 
 if os.environ.get("CIRCLECI"):
     from .ci import *  # noqa
+
+
+# Register Rich as default handler for stacktraces
+from rich.traceback import install
+
+install(show_locals=True)
```

Comment:
Using Rich to handle stacktraces like this give you a bunch of extra info which massively improves the usefulness of error messages.

**Default:**
![Screenshot at 2021-08-13 21-15-10](https://user-images.githubusercontent.com/6025893/129415752-4d9dc3c8-708f-467a-bcc9-24244b902db4.png)

**Rich:**
![Screenshot at 2021-08-13 21-14-44](https://user-images.githubusercontent.com/6025893/129415770-90126b63-c6b0-478f-a723-16e22634fedb.png)

Reply:
This looks ace!